### PR TITLE
wicked: Add test case "teaming roundrobin"

### PR DESCRIPTION
**Pull request by cfconrad**
Introduce new test for teaming:

```
@teams
Scenario: Teaming, roundrobin
When I team together eth0 and eth1 in roundrobin mode
Then there should be a new team0 card
And both cards should be enslaved to team0
And I should be able to ping the other side of the aggregated link
```

- Related ticket: https://progress.opensuse.org/issues/50909
- Verification run: 
  - http://cfconrad-vm.qa.suse.de/tests/4120 (SUT)
  - http://cfconrad-vm.qa.suse.de/tests/4119 (REF)

### CODE REVIEW

File: data/wicked/teaming/ifcfg-team0-roundrobin [hunk 1]
```diff
@@ -0,0 +1,10 @@
+STARTMODE=auto
+BOOTPROTO=static
+
+TEAM_RUNNER=roundrobin
+TEAM_PORT_DEVICE=eth0
+TEAM_PORT_DEVICE_1=eth1
+
+#IPADDR='192.168.50.123/24'
```
Comment (asmorodskyi) [inline data/wicked/teaming/ifcfg-team0-roundrobin]: commented line , needs to be removed

File: data/wicked/teaming/ifcfg-team0-roundrobin [hunk 2]
```diff
@@ -0,0 +1,10 @@
+STARTMODE=auto
+BOOTPROTO=static
+
+TEAM_RUNNER=roundrobin
+TEAM_PORT_DEVICE=eth0
```
Comment (asmorodskyi) [inline data/wicked/teaming/ifcfg-team0-roundrobin]: we should 'sed' it at runtime from iface value , no hardcoding

File: data/wicked/teaming/ifcfg-team0-roundrobin [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+TEAM_PORT_DEVICE_1=eth1
```
Comment (asmorodskyi) [inline data/wicked/teaming/ifcfg-team0-roundrobin]: we should 'sed' it at runtime from iface value , no hardcoding

File: data/wicked/teaming/ifcfg-team0-roundrobin [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+#IPADDR='192.168.50.123/24'
+IPADDR='10.0.2.11/15'
```
Comment (asmorodskyi) [inline data/wicked/teaming/ifcfg-team0-roundrobin]: we should 'sed' it at runtime from iface value , no hardcoding

File: tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Teaming, roundrobin
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
```
Comment (asmorodskyi) [inline tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm]: we already had this discussion I think :)

File: tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    $self->get_from_data('wicked/teaming/ifcfg-eth0',             $cfg_ifc0);
+    $self->get_from_data('wicked/teaming/ifcfg-eth1',             $cfg_ifc1);
+    $self->get_from_data('wicked/teaming/ifcfg-team0-roundrobin', $cfg_team0);
+
+    $self->wicked_command('ifup', 'team0');
+    ifc_exists('team0');
+    validate_script_output('ip a s dev ' . $ctx->iface(),  sub { /master team0/ });
+    validate_script_output('ip a s dev ' . $ctx->iface2(), sub { /master team0/ });
+
+    $self->ping_with_timeout(type => 'host', interface => 'team0');
```
Comment (asmorodskyi) [inline tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm]: this two lines , copy/pasted in 3 tests up to now ( this one + 2 flavors of aggregate both cards ) maybe it is the time to extract it to a function ?

File: tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm (hunk 5 above)
Comment (cfconrad) [reply tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm]: Yes but now I do not change it, but make it at a new file :)

File: tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm (hunk 6 above)
Comment (cfconrad) [reply tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm]: Hm, I think you mix it with `SLAVE` for binding.
But yes, we could make it as a utility function.

File: tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm (hunk 5 above)
Comment (asmorodskyi) [reply tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm]: ok

File: tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm (hunk 6 above)
Comment (cfconrad) [reply tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm]: Sorry I misunderstood you. I thought you are speaking about the `ip a s dev | grep master` lines.
Put the code inside of `ping_with_timeout()`

**Comment by cfconrad** [comment]
VR: http://cfconrad-vm.qa.suse.de/tests/4132

**Comment by asmorodskyi** [comment]
gut gemacht :)

**Pull request merged**